blueprints/gcve/monitoring/README.md

```diff
@@ -0,0 +1,117 @@
+
+<p align="center">
+  <img src="gcve-mon-diagram.png" alt="GCVE Logging and Monitoring Blueprint">
+</p>
+
+## Deployed Resources
+
+This blueprint deploys and configures the following resources:
+
+* **Service Account:** Grants the Bindplane agent permissions to write logs/metrics and access Secret Manager.
+* **Firewall Rule (optional):** Allows health checks on TCP port 5142 to ensure the agent is running.
+* **Monitoring Dashboards (optional):** Provides default dashboards for GCVE metrics.
+* **VM Template:** Creates a Debian 11-based template for the Bindplane agent.
+* **Managed Instance Group:** Manages the deployment and provides autohealing to the Bindplane agent.
+* **Secret Manager Secrets:** Stores vCenter credentials (username, password, FQDN).
+
+## Completing the Setup
+
+After deploying this blueprint, you need to complete the following steps:
+* [Configure GCVE to send traffic to the Bindplane agent](https://cloud.google.com/vmware-engine/docs/environment/howto-forward-syslog), which listens on TCP port 5142 by default.
+* Update secrets in Secret Manager with vCenter credentials and FQDN.
+
+## Troubleshooting
+
+If you encounter issues, check the following:
+
+* **Firewall:** Ensure that the firewall rule allows traffic to TCP port 5142.
+* **vCenter Configuration:** Verify that GCVE is correctly configured to forward syslog messages.
+* **Agent Logs:** Examine the Bindplane agent logs for errors.
+
+## Security Considerations
+
+* **Least Privilege:** Grant the Bindplane agent service account only the necessary permissions.
+* **Secret Management:** Store vCenter credentials securely in Secret Manager.
+
+<!-- BEGIN TOC -->
+- [Overview](#overview)
+- [Deployed Resources](#deployed-resources)
+- [Completing the Setup](#completing-the-setup)
+- [Troubleshooting](#troubleshooting)
+- [Security Considerations](#security-considerations)
+- [Basic Monitoring setup with default settings](#basic-monitoring-setup-with-default-settings)
+- [Variables](#variables)
+- [Outputs](#outputs)
+<!-- END TOC -->
+
+## Basic Monitoring setup with default settings
+
+```hcl
+
+module "gcve-monitoring" {
+  source     = "./fabric/blueprints/gcve/monitoring"
+  project_id = "gcve-mon-project"
+  project_create = {
+    billing_account = "0123AB-ABCDEF-123456"
+    parent          = "folders/1234567890"
+    shared_vpc_host = "abcde-prod-net-spoke-0"
+  }
+
+  vm_mon_config = {
+    vm_mon_name = "bp-agent"
+    vm_mon_type = "e2-small"
+    vm_mon_zone = "europe-west1-b"
+  }
+
+  vpc_config = {
+    host_project_id      = "abcde-prod-net-spoke-0"
+    vpc_self_link        = "https://www.googleapis.com/compute/v1/projects/abcde-prod-net-spoke-0/global/networks/prod-spoke-0"
+    subnetwork_self_link = "projects/abcde-prod-net-spoke-0/regions/europe-west1/subnetworks/prod-default-ew1"
+  }
+
+  vsphere_secrets = {
+    secret_vsphere_server   = "gcve-mon-vsphere-server"
```

Comment:
nit: I would drop the `secret_` in front of each attribute, since they're all part of `vsphere_secrets`

Reply:
makes sense... updated.